test/hip_device.cpp

```diff
@@ -104,6 +105,9 @@ namespace rocwmma
         CHECK_RSMI_ERROR(rsmi_dev_gpu_clk_freq_get(m_smiDeviceIndex, RSMI_CLK_TYPE_SYS, &freq));
```

Comment:
Which API is returning the 'UNSUPPORTED' code?
Might want to handle the return code instead of deferring only to benchmark mode.